apps/mobile/utils/clipboard.ts

```
@@ -0,0 +1,35 @@
+/* eslint-disable no-console */
```

We don't want console logs.
For now we can just ignore the error
Just catch it and do nothing:

```
catch (_error) {
  //
}
```

---

Got it. We should probably work towards adding an error logger. Fixed https://github.com/satsigner/satsigner/pull/83/commits/9f8670248e3992ab78a023a6df948794f43d6d6c